### PR TITLE
feat: add FFI function to create seed words from Mbase58 encrypted cipher seed

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
add FFI function to create seed words from Mbase58 encrypted cipher seed

Motivation and Context
---
Mobile clients need to get the seed words out so that they can recover the wallet based on an encrypted cipher seed. 

How Has This Been Tested?
---
unit test

### CODE REVIEW

File: applications/minotari_console_wallet/src/automation/commands.rs [hunk 1]
```diff
@@ -2485,15 +2486,36 @@ pub async fn command_runner(
...
+                                .map_err(|e| CommandError::General(e.to_string()))?;
+                            CipherSeed::from_enciphered_bytes(&bytes, passphrase)
+                                .map_err(|e| CommandError::General(e.to_string()))?
+                        },
+                        (_, _) => {
+                            return Err(CommandError::General(
+                                "Either seed words or cipher seed must be provided".to_string(),
+                            ))
+                        },
+                    };
```
Comment (hansieodendaal) [inline applications/minotari_console_wallet/src/automation/commands.rs]: I think the true and false in both cases should be mirrored. For example, if `args.seed_words.as_str().is_empty()` then `SeedWords::from_str(args.seed_words.as_str())` will fail.

**Review by hansieodendaal — COMMENTED** [review]
One comment

File: applications/minotari_console_wallet/src/automation/commands.rs (hunk 1 above)
Comment (SWvheerden) [reply applications/minotari_console_wallet/src/automation/commands.rs]: good catch here

**Review by hansieodendaal — APPROVED** [review]
utACK

**Pull request merged**